src/app/components/ReadTime/index.tsx

```diff
@@ -145,28 +149,36 @@ export const ReadTime = ({
 
   if (!validRender) return null;
 
-  const { readTimeInMilliseconds, minutesLabel, copy } = ProcessReadTime({
+  const { readTimeInMilliseconds, copy } = ProcessReadTime({
     readTimeValue: readTimeValue as number,
     readTimeVariant: readTimeVariant as string,
   });
 
-  const eventTrackingData: EventTrackingData = {
+  const optimizelyTrackingData: EventTrackingData = {
     componentName: 'read-time',
     sendOptimizelyEvents: true,
     experimentName: 'newswb_ws_homepage_read_time',
     experimentVariant: readTimeVariant,
+  };
+
+  const eventTrackingData: EventTrackingData = {
+    ...optimizelyTrackingData,
     itemTracker: {
-      label: `Read time: ${readTimeValue} ${minutesLabel}`,
+      type: promoType,
+      position: promoPosition,
+      label: `Read time: ${readTimeValue} ${readTimeValue === 1 ? 'minute' : 'minutes'}`,
       duration: readTimeInMilliseconds,
       resourceId: promoId,
     },
   };
 
-  // eslint-disable-next-line react-hooks/rules-of-hooks
-  const viewRef = useViewTracker(eventTrackingData);
-
   const isControlVariant = readTimeVariant === 'control';
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const viewRef = useViewTracker(
+    isControlVariant ? optimizelyTrackingData : eventTrackingData,
```

Comment:
Made this change so that the read time data is not included in the control variant. Just makes it clearer that in the control variant, the user doesn't actually see the read time.